plugins/com.google.cloud.tools.eclipse.util.test/src/com/google/cloud/tools/eclipse/util/CloudToolsInfoTest.java

```diff
@@ -24,4 +24,9 @@
   public void testGetToolsVersion() {
     Assert.assertTrue(CloudToolsInfo.getToolsVersion().startsWith("0.1.0."));
   }
+
+  @Test
+  public void testUserAgent() {
+    Assert.assertTrue(CloudToolsInfo.USER_AGENT.startsWith("gcloud-eclipse-tools/0.1.0"));
```

Comment:
https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/pull/468#discussion_r72632320

The value is actually something like `gcloud-eclipse-tools/0.1.0.201608160330` (if ran with `mvn verify` on the command-line). For this reason, the other test `testGetToolsVersion()` also uses `startsWith()`.